gradle/libs.versions.toml

```diff
@@ -4,6 +4,8 @@ org-jetbrains-compose = "1.2.1"
 org-jetbrains-kotlin = "1.7.20"
 com-android-tools-build = "7.2.2"
 com-squareup-sqldelight = "1.5.3"
+io-gitlab-arturbosch-detekt = "1.22.0-RC2"
```

Comment:
is this latest?

Reply:
Just checked again and it looks like there's RC3 released 3 days ago. Will update to that